src/classes/FORM_ServiceGiftEntry.cls

```diff
@@ -174,20 +174,38 @@ public with sharing class FORM_ServiceGiftEntry {
     *
     * @param id: Form_Template__c record id
     *
-    * @return list: List of deleted Form_Template__c names
+    * @return FormTemplateWrapper: Wrapper object of the list of deleted template names and the result
+    * of the DML action
     */
     @AuraEnabled
-    public static String[] deleteFormTemplates(String[] ids) {
-        Form_Template__c[] templates = [SELECT Id, Name FROM Form_Template__c WHERE Id IN :ids];
-        delete templates;
-
+    public static FormTemplateWrapper deleteFormTemplates(String[] ids) {
         String[] formTemplateNames = new String[] {};
-        for (Form_Template__c template : templates) {
-            formTemplateNames.add(template.Name);
+        Form_Template__c[] templates = [
+                SELECT Id, Name
+                FROM Form_Template__c
+                WHERE Id IN :ids
+        ];
+
+        FormTemplateWrapper templateWrapper;
+
+        try {
+            delete templates;
```

Comment:
Absolutely valid, the `handleError` method looks to be able to process the error, although the mode for the toast will be 'sticky'.

Reply:
Wanted to make a quick suggestion. We might be able scrub this entire delete method from `FORM_ServiceGiftEntry` by importing the delete method from `lightning/uiRecordApi`.

I think it would a straight swap out of the delete method being imported from `FORM_ServiceGiftEntry` for the platform provided one. Starting to use this in the Batch Wizard work. Linked methods I'm importing from `uiRecordApi` below and to the delete method docs.

https://github.com/SalesforceFoundation/NPSP/pull/4959/files#diff-1d0efebf882c6e1dd1bd1089e48aec0cR11

https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.reference_delete_record

Reply:
Thanks @psadie-sfdo! , It's super clean, simple and it works. Using it for the first time.

Reply:
👍🏽